packages/compiler/src/app/builders/blueprints/.angular-cli.blueprint.json

```diff
@@ -17,13 +17,13 @@
       "tsconfig": "tsconfig.json",
       "prefix": "@@name",
       "styles": [
-        "styles.css"
+        "styles.scss"
```

Comment:
We should make sure the user can still use css if they want to

Reply:
In the compiler, I had it put both css and scss code into an scss file. In this way, the compiled app will only have scss and no css. 

I could add .css here too but I didn't for this commit.

Reply:
Ohh I see, that's clever. Can you add a comment in the code where you do that to make it more clear?

Reply:
Added in `compiler/src/app/builders/app.compiler.ts`!